ci-operator/templates/openshift/installer/cluster-launch-installer-libvirt-e2e.yaml

```diff
@@ -188,20 +191,44 @@ objects:
         #!/bin/bash
         set -euo pipefail
 
-        trap 'rc=$?; if test "${rc}" -eq 0; then touch "${HOME}"/setup-success; else touch "${HOME}"/exit; fi; exit "${rc}"' EXIT
+        trap 'rc=$?; if test "${rc}" -eq 0; then touch "${HOME}"/setup-success; else touch "${HOME}"/exit "${HOME}"/setup-failed; fi; exit "${rc}"' EXIT
         trap 'CHILDREN=$(jobs -p); if test -n "${CHILDREN}"; then kill ${CHILDREN} && wait; fi' TERM
         cat  > "${HOME}"/run-tests.sh << 'EOF'
         #!/bin/bash
         set -euo pipefail
         export PATH=/home/packer:$PATH
         mkdir -p /tmp/artifacts/junit
 
+        # this is a hack to extend the install time allowed by installer
+        # TODO: remove this
+        # https://github.com/openshift/installer/issues/3043
+        export KUBECONFIG=/home/packer/clusters/installer/auth/kubeconfig
+        attempts=0
+        while (( attempts++ < 40 )); do
+          if [[ $attempts -eq 39 ]]; then
+            set -x
+            oc get co
+            set +x
+            echo "cluster install failed, exiting"
+            exit 1
+          fi
+          set +eu
+          oc get co | awk '{ print $3 }' | grep False
+          if [[ "$?" -ne 0 ]]; then
+            echo "all clusteroperators now available"
+            break
+          fi
+          echo "waiting up to 20 minutes for install to finish..."
+          sleep 30
+        done
+
```

Comment:
this setup launches a nested libvirt in gcp, it's expected to take longer - perhaps someone from installer/crc can work on this so this hack can be removed - from what I've seen it's usually ~5 min over the timeout, usually the installer times out at 99% or 100% complete.

Reply:
@sallyom Should we create an issue on installer side for same?

Reply:
@praveenkumar I added to this issue: https://github.com/openshift/installer/issues/3043#issuecomment-602702473  and opened/closed this PR: https://github.com/openshift/installer/pull/3333  installer team wants someone (not me as i don't have the time right now) to investigate why installs w/ libvirt are slow - but this setup is a nested libvirt install so I believe it's understandable that it's especially slow.

Reply:
@sallyom Sounds good to me but two comments:

* There is already `wait-for install-complete` subcommand in the installer, can't we just use that instead?
* Would be good to have your [comment here](https://github.com/openshift/release/pull/7568/commits/ee2311973de224a1cf8e91d05d2b7c7e3f132ea7#r398123726) in the commit log itself.

Reply:
@zeenix oh? I did not know of the subcommand, I'll try that instead, that will be much better.  Is that subcommand documented? thanks.

Reply:
I don't know but `openshift-install wait-for --help` will give you some info. It just tell the Installer to wait another 30 mins.

Reply:
calling `wait-for install-complete` a second time is the way to go.

Reply:
sweeeet thanks will make the change

Reply:
let's all note that that does work, however :laughing:

Reply:
done, I added it to the create-cluster script that runs in this template.  The create-cluster script is here: https://github.com/ironcladlou/openshift4-libvirt-gcp/blob/0e9ee53c77abcef554dc3e1a76d1adbbd414962a/tools/create-cluster#L68-#L73

Reply:
> Commit log:
> 
> * typo: "instead" -> "instead of"
> * Could you please summarize a bit why this change is needed (in the commit log)?
> 
> **UPDATED:** Forgot stupid GH still doens't know about per-patch reviews. This was about the `[libvirt-e2e] Use custom subnet instead auto for GCP` patch.

@praveenkumar this commit is yours, and honestly i'm hazy on why the custom subnet instead of auto - if you can add a bit about that here I'll add it to the commit msg.

Reply:
Updated the comment https://github.com/openshift/release/pull/7572/commits